### PR TITLE
Fix 404 pages

### PR DESCRIPTION
**Pull request by harrison**
screen shot 2015-05-21 at 11 26 55

### CODE REVIEW

**Comment by harrison** [comment]
hm this is pretty ghetto.. this is only when hitting a 404 page in JS, which should pretty much never happen. In prod the only 404 page is the one set by s3, so you can never get a translated 404 page.. only way to really deal with this is run the express server in prod:(

@english thoughts?

File: app/pages/not-found/not-found.js [hunk 1]
```diff
@@ -1,14 +1,33 @@
 import React from 'react';
-import Page from '../../components/page/page';
+import Message from '../../components/message/message';
+
+if (process.env.BROWSER) {
+  require('./not-found.css');
```
Comment (english) [inline app/pages/not-found/not-found.js]: What's going on here?

Comment (english) [inline app/pages/not-found/not-found.js]: Is it not bundled up with the rest of the app?

Comment (harrison) [inline app/pages/not-found/not-found.js]: thought it wasn't but is, moving to main

**Comment by english** [comment]
Hmmm. That's a good point about only being able to show English 404s without the web server. One to think about, but it doesn't seem like a such a big deal...

**Comment by english** [comment]
What exactly is the scenario for hitting a JS 404?

**Comment by harrison** [comment]
In development and when loading the site in an iframe or proxied where the browser url is something like `googlecache.com?gocardless.com`

**Comment by english** [comment]
Ah ok, cool. :+1:

**Pull request merged**